client/my-sites/site-settings/manage-connection/index.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import HeaderCake from 'components/header-cake';
+import Main from 'components/main';
+import SiteOwnership from './site-ownership';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+
+class ManageConnection extends Component {
+	componentDidMount() {
+		this.verifySiteIsJetpack();
+	}
+
+	componentDidUpdate() {
+		this.verifySiteIsJetpack();
+	}
+
+	verifySiteIsJetpack() {
+		if ( this.props.siteIsJetpack === false ) {
+			this.redirectToGeneral();
+		}
+	}
+
+	redirectToGeneral = () => {
+		const { siteSlug } = this.props;
+
+		page( '/settings/general/' + siteSlug );
+	};
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<Main className="manage-connection site-settings">
+				<DocumentHead title={ translate( 'Site Settings' ) } />
+
+				<HeaderCake onClick={ this.redirectToGeneral }>
+					{ translate( 'Manage connection' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Manage Connection' )` *ES Score: 18*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-settings-manage-connection-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).